### PR TITLE
Update entities special cases

### PR DESCRIPTION
**Pull request by atothhpe**
In the end, all I did was UI fixes. I would like to separate the tree stuff in a separate branch, since it's a lot of crap

### CODE REVIEW

File: octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java [hunk 1]
```diff
@@ -193,25 +199,38 @@ private void drawEntityFields(Set<String> shownFields, EntityModelWrapper entity
...
             Control fieldEditorControl = (Control) fieldEditor;
             fieldEditorControl.setLayoutData(fieldEditorGridData);
             fieldEditorControl.setForeground(foregroundColor);
 
-            //In case of uneven number of fields, the row heights would be inconsistent
-            //and Eclipse does not know how to scale correctly
+            // In case of uneven number of fields, the row heights would be
+            // inconsistent
+            // and Eclipse does not know how to scale correctly
             if (i == shownFields.size() - 1 && i % 2 != 1) {
```
Comment (paul-ciobanu) [inline octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java]: @atothhpe @vladbenchea I know that you didn't write this, but i % 2 != 1 doesn't seem right.
Perhaps you can update this while it is still fresh in our minds to take into account the skipped fields (line 186 from above)

File: octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/BooleanFieldEditor.java [hunk 2]
```diff
@@ -12,66 +12,83 @@
...
-                String text = combo.getText();
-                Boolean bool = Boolean.parseBoolean(text);
-                entityModelWrapper.setValue(new BooleanFieldModel(fieldName, bool)); 
+            public void widgetSelected(SelectionEvent e) {
+                boolean newValue;
+                if(booleanEntityComboBox.getSelectedEntity() == ENTITY_TRUE) {
+                    newValue = true;
+                } else {
+                    newValue = false;
+                }
```
Comment (paul-ciobanu) [inline octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/BooleanFieldEditor.java]: @atothhpe What about boolean newValue = booleanEntityComboBox.getSelectedEntity() == ENTITY_TRUE; ? :)

File: octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java (hunk 1 above)
Comment (atothhpe) [reply octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/EntityFieldsComposite.java]: hahahahaha

File: octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/BooleanFieldEditor.java (hunk 2 above)
Comment (atothhpe) [reply octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/BooleanFieldEditor.java]: :D

Comment (atothhpe) [reply octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/BooleanFieldEditor.java]: good point

**Pull request merged**